css/project-page.styl

```diff
@@ -1,3 +1,21 @@
+.about-container
+  background: rgba(255,255,255,0.95)
+  border-radius: 3px
+  color: #000000
+  margin: 1.5em 12vw
+  padding: 1.5em 6vw
+
+  img
+    max-width: 100%
+
+  h1:first-child, h2:first-child,
+  h3:first-child, h4:first-child,
+  h5:first-child, h6:first-child
+    color: #f48a21
```

Comment:
Are these colors from the upcoming redesign? Perhaps move them to `common.styl` and give them a variable name so they're easily reusable.